pyccel/errors/errors.py

```diff
@@ -1,3 +1,8 @@
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details.     #
+#------------------------------------------------------------------------------------------#
+# pylint: disable=missing-module-docstring
```

Comment:
Please do not disable missing docstrings outside of the tests/ folder.

This file has a clear enough purpose for a docstring to be written